src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -57,7 +57,7 @@
     "priority": "nil"
   } ],
   "sellers" : [ {
-    "name" : "Alice Pauline",
+    "name" : "SAlice Pauline",
```

Comment:
Thanks for fixing the adherence to our convention of sellers starting with an S.